tests/pipeline/test_swab_deltas.py

```diff
@@ -0,0 +1,33 @@
+import pandas as pd
+import pytest
+from mimesis.schema import Field
+from mimesis.schema import Schema
+
+from cishouseholds.pipeline.swab_delta_ETL import transform_swabs_delta
+
+_ = Field("en-gb", seed=42)
+
+
+@pytest.fixture
+def input_df(spark_session):
+    """
+    Generate lab swabs file.
+    """
+    lab_swabs_description = lambda: {  # noqa: E731
```

Comment:
Can we run the test without the lambda, just assigning the dict to the variable.